src/core/functions/scalar/llm_embedding.cpp

```diff
@@ -19,17 +19,17 @@ static void LlmEmbeddingScalarFunction(DataChunk &args, ExpressionState &state,
     auto model_details_json = CoreScalarParsers::Struct2Json(args.data[1], 1)[0];
     auto model_details = ModelManager::CreateModelDetails(con, model_details_json);
 
-    auto embeddings = nlohmann::json::array();
+    vector<string> prepared_inputs;
     for (auto &row : inputs) {
```

Comment:
`prepared_inputs` is not a clear name, let's find a better name, you can also provide the input size for prepared_inputs in its ctor.